api/src/TemplateParser.php

```diff
@@ -20,6 +20,8 @@ class TemplateParser
         var $params = array();
 
         function __construct($db) {
+
+            print "In BaseClass constructor\n";
```

Comment:
```suggestion
```

Reply:
Confused as not part of my commit but will commit anyway..